code/items/ammo/TTTAmmo.cs

```diff
@@ -0,0 +1,92 @@
+using System;
+using Sandbox;
+using TTTReborn.Player;
+
+namespace TTTReborn.Items
+{
+    [Library("ttt_ammo"), Hammer.Skip]
+    public abstract partial class TTTAmmo : Prop
+    {
+        /// <summary>
+        /// String definition of ammo type, should match TTTWeapon.AmmoType
+        /// </summary>
+        public virtual string AmmoType { get; set; }
```

Comment:
If we are going to stick with `string`, I feel like `AmmoName` is a better name.

Reply:
50-50. I think Type works better since the entity definition is for the physical "ammo box" and it's more so picking the "type" of ammo it holds, rather than the "name" of it.
Up for consideration since the implementations are technically the only data pieces we have for certain ammo types, such as `AmmoMax` being the maximum amount of ammo a player can carry.

Reply:
I think we actually use `Name` for when referencing `Perks`, check out `TTTPerk.cs`. There might be two things here to consider

1. We might want to rename to `AmmoName`
2. We might want to drop the `Ammo` prefix as well, since people will be referencing these params as `ammo.AmmoType`

Thoughts @Alf21

Reply:
Agreed with last mentioned hint: `Name` should be the way to go, as references gonna be `ammo.Name`

Reply:
Reasonable to me. Will get this all pushed in.